### PR TITLE
feat: adds inngest workshop

### PR DESCRIPTION
**Pull request by dimitropoulos**
## Overview

Adds Inngest Workshop.

💖

### CODE REVIEW

File: src/data/notion-data.ts [hunk 1]
```diff
@@ -40,6 +40,25 @@ export const peopleBySlug: Record<string, Person> = {
 		role: "Open Source",
 		slug: "aakansha-doshi",
 	},
+	"aaron-harper": {
+		bio: "I’m a software developer in Ann Arbor, Michigan. I’m a fan of writing reliable code, my dog Benny, and attending my favorite local meetup: Michigan TypeScript.  When I’m not helping developers implement durable workflows, I love spending time with my wife and daughter, gardening, and building orchestration engines that combine event driven architecture and workflows.",
```
Comment (JoshuaKGoldberg) [inline src/data/notion-data.ts:44]:
```suggestion
		bio: "I’m a software developer in Ann Arbor, Michigan. I’m a fan of writing reliable code, my dog Benny, and attending my favorite local meetup: Michigan TypeScript. When I’m not helping developers implement durable workflows, I love spending time with my wife and daughter, gardening, and building orchestration engines that combine event driven architecture and workflows.",
```

File: src/pages/speakers.astro [hunk 2]
```diff
@@ -6,11 +6,6 @@ import { peopleBySlug, talksBySlug } from "~/data/notion-data";
 import Speaker from "~/components/Speaker.astro";
 import BodyText from "~/components/BodyText.astro";
 import SafeMarkdown from "~/components/SafeMarkdown.astro";
-
-const speakersToExclude = new Set([
-	// Sylwia might not be able to make it 😢
-	"sylwia-vargas",
-]);
```
Comment (JoshuaKGoldberg) [inline src/pages/speakers.astro:13]: Sad to see the - on Sylwia, but happy to see the removal of code logic in general!

File: src/pages/badge/front.astro [hunk 3]
```diff
@@ -17,7 +17,7 @@ const role = Astro.url.searchParams.get("role");
 
 <BadgeLayout>
 	<div class="contents">
-		<div class="more-top-gr3adient"></div>
+		<div class="more-top-gradient"></div>
```
Comment (JoshuaKGoldberg) [inline src/pages/badge/front.astro:20]: Lol, is this unused code?

**Review by JoshuaKGoldberg — APPROVED** [review]
🔥

File: src/pages/badge/front.astro (hunk 3 above)
Comment (dimitropoulos) [reply src/pages/badge/front.astro:20]: lol no idea, it was there for a bit, it seems

File: src/data/notion-data.ts (hunk 1 above)
Comment (dimitropoulos) [reply src/data/notion-data.ts]: lol.  fixed in f9f5e938a2905d86dea37642a0c2916b8b374c98

**Pull request merged**